modules/migration-using-must-gather.adoc

```diff
@@ -4,13 +4,21 @@
 // * migration-toolkit-for-containers/troubleshooting-mtc
 
 [id="migration-using-must-gather_{context}"]
-= Using must-gather to collect data
 
-You must run the `must-gather` tool if you open a customer support case on the link:https://access.redhat.com[Red Hat Customer Portal] for the {mtc-full} ({mtc-short}).
 
-The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migration-specific logs and data that are not collected by the default `must-gather` image.
 
-.Procedure
+= Collecting logs and metrics with the must-gather tool
+
+You can collect logs, metrics, and information about {MTC-short} custom resources by using the `must-gather` tool.
+
+You can view metrics data with Prometheus, a component of the OpenShift Container Platform monitoring stack.
```

Comment:
Use the correct attribute for OCP. You can find it in common-attributes.adoc

Reply:
The attribute for OCP is not in common-attributes.adoc. Also, the name is written out in full here. Should it be an attribute?

Reply:
No. Do not add attributes to OCP docs unless you clear it with the DPM.

Reply:
So where do I get the attribute for OCP?

Reply:
It's probably just {product-title}.